addons/account/models/account.py

```diff
@@ -1515,6 +1515,64 @@ def _get_journal_balance(self, domain=None):
         balance, amount_currency = self._cr.fetchone()
         return amount_currency if journal_currency else balance
 
+    def _get_journal_outstanding_payments_account_balance(self, domain=None, date=None):
+        ''' Get the outstanding payments balance of the current journal by filtering the journal items using the
+        journal's accounts.
+
+        /!\ The current journal is not part of the applied domain. This is the expected behavior since we only want
```

Comment:
here however, I'm less sure: the outstanding payment/receipt account will probably be shared among all journals.

@avw-odoo can you confirm?

Reply:
as discussed: keep it like that, but ensure an account set as outstanding payment/receipt or bank account can only be set on a single journal (via a constraint)